Formula/plotjuggler.rb

```diff
@@ -0,0 +1,35 @@
+class Plotjuggler < Formula
+  desc "Visualize time series that is fast, powerful and intuitive"
+  homepage "https://github.com/facontidavide/PlotJuggler"
+  url "https://github.com/facontidavide/PlotJuggler.git", tag: "3.7.1", revision: "fb2bab446f63c726f85e71b155d9fcbeb5603dc1"
```

Comment:
```suggestion
  url "https://github.com/facontidavide/PlotJuggler.git",
      tag: "3.7.1",
      revision: "fb2bab446f63c726f85e71b155d9fcbeb5603dc1"
```

Reply:
Do we need a Git checkout? Can we use a tarball instead?

Reply:
GitHub has a tarball tied to the tag, but I found other Formulas that reference GitHub's tag and hash, so I did the same.
Would a GitHub tarball be more ideal?

Reply:
Yes, it caches much better